bert_e/git_host/github/__init__.py

```diff
@@ -624,7 +609,10 @@ def remove_unwanted_workflows(self):
         self._check_suites = list(map(lambda elem: {
             **elem,
             'html_url':
-                dispatched.get(elem['id'], {'html_url': ''})['html_url']
+                dispatched.get(elem['id'], {
+                    'html_url':
+                        f"https://github.com/{self.repo}/commit/{self.commit}"
+                })['html_url']
         }, self._check_suites))
```

Comment:
You don't need this, and this make this function `self._get_aggregate_workflow_dispatched`   only get `dispatch` event. And just get the ID to remove dispatched `check-suite`. 
This should remove like ~20 to 30 lines of dead code

Reply:
should be good now